src/odemis/gui/cont/streams.py

```diff
@@ -544,6 +548,28 @@ def _on_metadata_btn(self, evt):
                         v = u"[%s … %s]" % (u", ".join(str(a) for a in v[:20]), u", ".join(str(a) for a in v[-20:]))
                     text += u"%s: %s\n" % (key, v)
 
+        # only display extra settings once
+        if model.MD_EXTRA_SETTINGS in raw[0].metadata:
+            text += u"\n======================================\nHardware Settings\n"
+            text += u"======================================\n"
+            for comp, vas in md[model.MD_EXTRA_SETTINGS].items():
+                if vas != {}:
+                    text += u"Component %s:\n" % comp
+                for name, value in vas.items():
```

Comment:
I think it's a little confusing that a "value" is a "data" + "unit". Maybe rename "value" to "content" and "data" to "value"?